typegate/tests/runtimes/prisma/prisma_test.ts

```diff
@@ -5,12 +5,14 @@ import { v4 } from "std/uuid/mod.ts";
 import { assert } from "std/assert/mod.ts";
 import { dropSchemas, recreateMigrations } from "../../utils/migrations.ts";
 import { gql, Meta } from "../../utils/mod.ts";
+import { randomSchema } from "../../utils/database.ts";
 
 Meta.test("prisma", async (t) => {
+  const schema = randomSchema();
   const e = await t.engine("runtimes/prisma/prisma.py", {
     secrets: {
       POSTGRES:
-        "postgresql://postgres:password@localhost:5432/db?schema=prisma",
+        `postgresql://postgres:password@localhost:5432/db?schema=${schema}`,
```

Comment:
maybe add a POSTGRES: randomSchemaPG() to be even more succinct?

Reply:
possible, will do that.

Reply:
Done.